outwatch/src/main/scala/outwatch/util/LocalStorage.scala

```diff
@@ -11,8 +11,8 @@ import org.scalajs.dom.window.{localStorage, sessionStorage}
 import outwatch._
 import outwatch.dom.dsl.events
 
-class Storage(domStorage: dom.Storage) {
-  private def subjectWithTransform(key: String, transform: Observable[Option[String]] => Observable[Option[String]])(implicit scheduler: Scheduler):IO[Handler[Option[String]]] = {
+class Storage[F[_]](domStorage: dom.Storage) extends ProHandlerOps[F] {
```

Comment:
I think the Sync constraint can be moved up here, so it doesn't have to be on every method.